examples/scripts/ct_fan_svmbir_ppp_bm3d_admm_prox.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+"""
+CT Reconstruction (ADMM Plug-and-Play Priors w/ BM3D, SVMBIR+Prox)
```

Comment:
Title should be different from that of the corresponding parallel beam example.

Reply:
Fixed it.